### ISSUE TITLE
PATH:COMBINE fails

### ISSUE THREAD

**Issue by tomekpiotrowski** [KSP-KOS/KOS#1685]
Fails with `failed to convert parameters`

**Comment by hvacengi** [comment]
Looking this over, it appears the error is in the casting of the parameters before passing them to the 
delegate:
```
[Log]: System.ArgumentException: failed to convert parameters
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in <filename unknown>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <filename unknown>:0 
  at System.Delegate.DynamicInvokeImpl (System.Object[] args) [0x00000] in <filename unknown>:0 
  at System.MulticastDelegate.DynamicInvokeImpl (System.Object[] args) [0x00000] in <filename unknown>:0 
  at System.Delegate.DynamicInvoke (System.Object[] args) [0x00000] in <filename unknown>:0 
  at kOS.Safe.Encapsulation.Suffixes.DelegateSuffixResult.Invoke (ICpu cpu) [0x00000] in <filename unknown>:0 
  at kOS.Safe.Compilation.OpcodeCall.StaticExecute (ICpu cpu, Boolean direct, System.Object destination, Boolean calledFromKOSDelegateCall) [0x00000] in <filename unknown>:0 
  at kOS.Safe.Compilation.OpcodeCall.Execute (ICpu cpu) [0x00000] in <filename unknown>:0 
  at kOS.Safe.Execution.CPU.ExecuteInstruction (IProgramContext context, Boolean doProfiling) [0x00000] in <filename unknown>:0 
```
So I think we're going to need to make the `VarArgsSuffix` accept all Structures, and then filter for string values.

**Comment by hvacengi** [comment]
Closed via #1746
